edx_sysadmin/git_import.py

```diff
@@ -366,6 +366,10 @@ def add_repo(repo, rdir_in=None, branch=None):
         # We want set course id in CourseGitLog as CourseLocator. So that in split module
         # environment course id remain consistent as CourseLocator instance.
         course_key = CourseLocator(*course_id)
+        update_outline_from_modulestore(course_key)
+        SignalHandler.course_published.send(
+            sender=course_key.course, course_key=course_key
+        )
```

Comment:
I know that the specific bug we are fixing is that course outlines and course published signals aren't getting sent, but I'm wondering if there is a more generic and robust solution. Does edx-platform have a simple import API (either python or REST) that we can use, and not have to worry about future changes to the import details?